### PR TITLE
Add autocomplete attribute on input and fix placeholder property.

### PR DESCRIPTION
**Pull request by dmaurya929**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/emailinput/v1/emailinput/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -60,6 +60,28 @@
                                                     jcr:primaryType="nt:unstructured"
                                                     sling:hideResource="{Boolean}true">
                                             </autocomplete>
+                                            <autocomplete-email
+                                                    wrapperClass="cmp-adaptiveform-textinput__autofillfieldkeyword"
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/emailinput/v1/emailinput/_cq_dialog/.content.xml:64]: this is email input `cmp-adaptiveform-emailinput`

File: ui.frontend/package-lock.json [hunk 2]
```diff
@@ -17817,7 +17822,8 @@
       "version": "8.8.0",
       "resolved": "https://registry.npmjs.org/ws/-/ws-8.8.0.tgz",
       "integrity": "sha512-JDAgSYQ1ksuwqfChJusw1LSJ8BizJ2e/vVu5Lxjq3YvNJNlROv1ui4i+c/kUUrPheBvQl4c5UbERhTwKa6QBJQ==",
-      "dev": true
+      "dev": true,
+      "requires": {}
```
Comment (rismehta) [inline ui.frontend/package-lock.json:17826]: Can you revert the changes in this file ?

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

**Comment by dmaurya929** [comment]
@sakshi-arora1 please review this PR.

**Review by sakshi-arora1 — APPROVED** [review]

**Pull request merged**